pkg/controllers/job/job_controller_actions.go

```diff
@@ -147,7 +147,8 @@ func (cc *jobcontroller) killJob(jobInfo *apis.JobInfo, podRetainPhase state.Pha
 	}
 
 	// Delete PodGroup
-	if err := cc.vcClient.SchedulingV1beta1().PodGroups(job.Namespace).Delete(context.TODO(), job.Name, metav1.DeleteOptions{}); err != nil {
+	pgName := job.Name + "-" + string(job.UID)
```

Comment:
This is not safe, the length could exceed k8s limit

Reply:
yep, thanks for your remind, so if I change the prefix into a constant such as 
1) "pod-group-"(it's from volcano-sh/apis's PodgroupNamePrefix), if uses it, the prefix unify.
2) or "job-pod-group-", add one new constant prefix into apis,  if uses it, we can distinguish job's podgroup from pod's podgroup easily.
so, which one do you think better?

Reply:
It is convinient that the podgroup name contains the job.Name. It is easy to know which job the podgroup belongs to without checking the ownereference.  From my point of view, job.Name + string(job.UID) is a better option. Besides, I am thinking whether the podgroup uid could use the Object ID generator util in #2145.

Reply:
i agree with your point about the podgroup name format, however, about the uid generator, i think it doesnot make much difference.

Reply:
similar to the pod name of Deployments :)